rendering/glb.ksy

```diff
@@ -0,0 +1,69 @@
+meta:
+  id: glb
+  title: GL Transmission Format, binary container
+  file-extension: glb
+  endian: le
+  license: MIT
+  xref:
+    mime: model/gltf-binary
+    wikidata: Q28135989
+
+doc: |
+  glTF is a format for distribution of 3D models optimized for being used in software
+
+doc-ref: https://github.com/KhronosGroup/glTF/tree/master/specification/2.0#binary-gltf-layout
+
+seq:
+  - id: header
+    type: header
+  - id: chunks
+    type: chunk
+    repeat: eos
+
+types:
+
+  header:
+    seq:
+      - id: magic
+        contents: glTF
+      - id: version
+        type: u4
```

Comment:
If I wanted to use this KSY spec for serialization, what I should set this field to in order to have the files recognized by other applications? Is there only one version that everybody uses, or several ones? How does the version affect the format structure/logic?

Please write this info into the `doc` key under this field.